e2e-tests/self-healing-advanced/run

```diff
@@ -4,26 +4,31 @@ set -o errexit
 set -o xtrace
 
 test_dir=$(realpath $(dirname $0))
-. ${test_dir}/../functions
+. "${test_dir}"/../functions
 cluster="self-healing-advanced"
 
 pumba_all_pods() {
     local action=$1
 
+    restartsBefore=$(kubectl get pod self-healing-advanced-pxc-0 -o yaml | grep restartCount: | awk '{print $2}')
     # run chaos for Pod
-    for i in $(seq 1 5); do
-        run_pumba $action "re2:$cluster-pxc-[0-9]+_"
+    for _ in $(seq 1 5); do
+        run_pumba "$action" "re2:$cluster-pxc-[0-9]+_"
     done
+
     sleep 60
+    
+    restartsAfter=$(kubectl get pod self-healing-advanced-pxc-0 -o yaml | grep restartCount: | awk '{print $2}')
 
-    kubectl_bin get pods \
-        | grep $cluster-pxc- \
-        | grep CrashLoopBackOff
+    # check if pod was restarted
+    if [ "$restartsAfter" -eq "$restartsBefore" ] ; then
+        exit 1
+    fi
```

Comment:
how do we know that became broken?
the previous logic was like - check if pods are definitely crashed.
new logic - just check that Pod have been restarted.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/337/commits/9c4f04794ac5663d22fcb90f4c41fcea02226fc1